app/views/shared/dialogs/_dialog_field.html.haml

```diff
@@ -31,7 +31,15 @@
       = render(:partial => "shared/dialogs/dialog_field_radio_button", :locals => locals)
 
     - when "DialogFieldDropDownList"
-      = render(:partial => "shared/dialogs/dialog_field_drop_down_list", :locals => locals)
+      - category_tags = field.values.collect(&:reverse)
+      - if field.multi_value?
+        = select_tag(field.name,
+                     options_for_select(category_tags, wf.value(field.name)),
+                     drop_down_options(field, url).merge(:multiple => true))
+        = javascript_tag "dialogFieldRefresh.initializeDialogSelectPicker('#{field.name}', '#{field.id}', undefined, '#{url}',
+        '#{field.trigger_auto_refresh}');"
+      - else
+        = render(:partial => "shared/dialogs/dialog_field_drop_down_list", :locals => locals)
```

Comment:
I think this whole if/else block should be within the `dialog_field_drop_down_list` partial, since it all relates to it. This file would remain unchanged.